phpBB/phpbb/textformatter/s9e/factory.php

```diff
@@ -395,6 +394,31 @@ public function regenerate()
 	}
 
 	/**
+	* Configure the Autolink / Autoemail plugins used to linkify text
+	*
+	* @param  \s9e\TextFormatter\Configurator $configurator
+	* @return void
+	*/
+	protected function configure_autolink(Configurator $configurator)
+	{
+		$configurator->plugins->load('Autoemail');
+		$configurator->plugins->load('Autolink', array('matchWww' => true));
+
+		// Create a tag that will be used to display the truncated text by replacing the original
+		// content with the content of the @text attribute
+		$tag = $configurator->tags->add('TRUNCATED_LINK');
```

Comment:
Probably not the best name for it. Maybe something like `AUTOLINK_TEXT` would make more sense.